updateQDKVersion.ps1

```diff
@@ -0,0 +1,61 @@
+$ver = $args[0]
+Write-Host $ver
+
+$csString = 'PackageReference Include=\"Microsoft\.Quantum\.[a-zA-Z\.]+\" Version=\"(?<oldVersion>[0-9|\.]+)\"'
+$csFiles = (Select-String -Path "**\*.csproj" -pattern "Microsoft.Quantum" | Select-Object -Unique Path)
+$csFiles | ForEach-Object {
+    (Get-Content -Encoding UTF8 $_.Path) | ForEach-Object {
+         $isQuantumPackage = $_ -match $csString
+         if ($isQuantumPackage) {
+             $_ -replace $Matches.oldVersion, $ver
+         } else {
+             $_
+         }
+    } | Set-Content -Encoding UTF8 $_.Path
+}
+
+$ipynb_string = '%package Microsoft.Quantum.Katas::(?<oldVersion>[0-9|\.]+)'
+$ipynbFiles =  (Select-String -Path "**\*.ipynb" -pattern "Microsoft.Quantum" | Select-Object -Unique Path)
+
+$ipynbFiles | ForEach-Object {
+    if ($_)
+    {
+        (Get-Content $_.Path) | ForEach-Object {
+            $isQuantumPackage = $_ -match $ipynb_string
+            if ($isQuantumPackage) {
+                $_ -replace $Matches.oldVersion, $ver
+            } else {
+                $_
+            }
+        } | Set-Content $_.Path
```

Comment:
Similarly here, I'd suggest explicitly denoting the encoding.

Reply:
Do you know what encoding the quantum operators are? I tried to change it to ascii or utf8 but it wouldn't translate those operators appropriately. @tcNickolas do you know?

Reply:
Let's omit the encoding parameter in the commands for the non-csproj files if omitting the encoding allows to preserve them unchanged.